HW6/Dockerfile

```diff
@@ -0,0 +1,29 @@
+
+FROM python:3.6-buster
+#Not using this for this assignment but I think
+#I will need it for the final so I left it
+#buster was required to get the mariadb library installed
+#should probably remove
+
+RUN apt-get update \
+    && apt-get -yy install libmariadb-dev
+
+# Install mysql client
+#not needed for this - help from Will
+RUN apt-get update
+RUN apt-get install -y mariadb-client
+
```

Comment:
Im not sure why everything is commented out, you need these lines to properly set up the docker container right?